addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -2259,31 +2259,29 @@ var BasicModel = AbstractModel.extend({
      *   context.
      * @param {Object} modifiers
      * @returns {Object}
+     * @throws {Error} if one of the modifier domains is invalid
      */
     _evalModifiers: function (element, modifiers) {
-        var result = {};
-        var self = this;
-        var evalContext;
-        function evalModifier(mod) {
+        var evalContext = null;
```

Comment:
you can (should) use const

Reply:
I'll just safely use ES4 everywhere. ES4ever!